src/pages/Check/index.jsx

```diff
@@ -13,12 +13,31 @@ const Check = () => {
   const history = useHistory()
   const { packages, loading, error } = useGetPackagesIdByCode(packageCode?.packageCode)
 
-  const toDelivery = async e => {
+  // eslint-disable-next-line complexity
+  const toDelivery = async (/** @type {{ packageCode: String; }} */ e) => {
     if (e?.packageCode && !error && !loading) {
-      const foundPackage = (await packages.length) > 0
+      const foundPackage = packages.length > 0
```

Comment:
Isn't needed the await anymore?

Reply:
Not needed as package length is not async

Reply:
Thanks